Meadowlark-js/docker-meadowlark/docker-compose.yml

```diff
@@ -37,9 +39,20 @@ services:
       DISABLE_SECURITY_DASHBOARDS_PLUGIN: true # disables security dashboards plugin in OpenSearch Dashboards
     networks:
       - opensearch-net
+    restart: unless-stopped
+
+  mongodb-node1:
+    image : mongo:4.0.28
```

Comment:
Chose MongoDB v4.0 because it is the latest version AWS DocumentDB is compatible with. From initial research, MongoDB 4.0 has a sufficient feature set meet our needs.

Reply:
I'm surprised they're compatible with 4.0, I had in back of my mind that this was the first version after license change.

Unfortunately, [Azure is only up to 3.6 if I'm reading correctly](https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb/feature-support-36#:~:text=is%20three%20years.-,Protocol%20Support,Cosmos%20DB's%20API%20for%20MongoDB.).

Reply:
Nope, just realized that they do have more than this, I just happened to land on an older page first :-).
https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb/feature-support-42

Reply:
Yeah, Cosmos DB is ahead of DocumentDB right now. https://docs.aws.amazon.com/documentdb/latest/developerguide/compatibility.html

Reply:
Interestingly, the DocumentDB defaults to 3.6 compatibility. You have to ask for 4.0 support. We'd figure it out pretty quick though because 4.0 is the first version to support transactions, and I'm pretty sure we're going to need them just like with DynamoDB.

Reply:
4.0 sounds good. I'll try this out as soon as possible.